src/dialogs/more-info/controls/more-info-timer.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  CSSResult,
+  css,
+  property,
+  PropertyValues,
+  customElement,
+} from "lit-element";
+import "@material/mwc-button";
+
+import { HomeAssistant } from "../../../types";
+import { TimerEntity } from "../../../data/timer";
+
+@customElement("more-info-timer")
+class MoreInfoTimer extends LitElement {
+  @property() public hass!: HomeAssistant;
+
+  @property() public stateObj?: TimerEntity;
+
+  protected render(): TemplateResult | void {
+    if (!this.hass || !this.stateObj) {
+      return html``;
+    }
+
+    return html`
+      <ha-attributes .stateObj="${this.stateObj}"></ha-attributes>
```

Comment:
Let's filter out `remaining`. It's confusing as it is not the actual remaining, but the remaining time when the state changed.

Reply:
Also let's drop quotes around dynamic expressions
```suggestion
      <ha-attributes .stateObj=${this.stateObj}></ha-attributes>
```